test/difficulty_index/example_difficulty_index.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env conda run -n diff_index_plot python
+# -*- coding: utf-8 -*-
+"""
+Load fieldijn from npz file created with save_ensemble_data.py
+helper function, compute ensemble mean and spread, compute
+difficulty index for a set of thresholds, plot and save the results.
+Created on Tue Mar 17 08:06:07 2020
+Last modified Mon Apr 6 11:30:30 2020
+@author: campbell
+
+Taken from original test_difficulty_index.py but replacing with METcalcpy and METplotpy.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from metcalcpy.calc_difficulty_index import forecast_difficulty as di
+from metcalcpy.calc_difficulty_index import EPS
+from metcalcpy.piecewise_linear import PiecewiseLinear as plin
+import metplotpy.difficulty_index.mycolormaps as mcmap
+from metplotpy.difficulty_index.plot_difficulty_index import plot_field

```

Comment:
like above, I could only successfully run if I used this import:
from difficulty_index.plot_difficulty_index import plot_field

Reply:
But when I run from command line and set the PYTHONPATH, it works as you originally set it.